ResidentInformationApi/serverless.yml

```diff
@@ -20,12 +20,17 @@ package:
 functions:
   residentInformationApi:
     name: ${self:service}-${self:provider.stage}
+    timeout: 15
```

Comment:
Is this a lambda timeout for 15s? Why 15?

Reply:
Yeh, unfortunately due to API going through to different APIs it takes longer than 6s, which makes sense so had to be extended

Reply:
Cool. 👍 
Yeah, that makes sense. Might be something to watch, especially if we start adding more APIs.